index.rst

```diff
@@ -48,11 +48,256 @@
+The latter is also important for the system image quality, but does not directly interact with the AOS, nor the support system.
+The same applies to the camera rotator (https://github.com/lsst-ts/ts_mtrotator), and the
+Vibration Monitoring System (VMS https://github.com/lsst-ts/ts_vms).
+The VMS has components on M1M3, M2, the camera rotator, and the Telescope Mount Assembly (TMA).
+
+The command structure and the naming of the variables are defined by the Interface Control Documents (ICDs): LTS-161 (M1M3), LTS-162 (M2), LTS-160 (hexapods) and MTAOS (LTS-163??).
+Their implementations are found in the XML:
+
+- M1M3 https://github.com/lsst-ts/ts_xml/tree/develop/sal_interfaces/MTM1M3
+- M2 https://github.com/lsst-ts/ts_xml/tree/develop/sal_interfaces/MTM2
+- M2 and Camera Hexapods https://github.com/lsst-ts/ts_xml/tree/develop/sal_interfaces/Hexapod
+- MTAOS https://github.com/lsst-ts/ts_xml/tree/develop/sal_interfaces/MTAOS
+
+Or, in more human-readable format:
+
+- M1M3 https://ts-xml.lsst.io/sal_interfaces/MTM1M3.html
+- M2 https://ts-xml.lsst.io/sal_interfaces/MTM2.html
+- M2 and Camera Hexapods https://ts-xml.lsst.io/sal_interfaces/MTHexapod.html
+- MTAOS https://ts-xml.lsst.io/sal_interfaces/MTAOS.html
+
+Each CSC also has its own documentation page:
+
+- M1M3 https://github.com/lsst-ts/ts_m1m3support/blob/master/README.md
+- M2 https://github.com/lsst-ts/ts_mtm2/blob/master/README.md
+- M2 and Camera Hexapods https://ts-mthexapod.lsst.io
+- MTAOS https://ts-mtaos.lsst.io
+
+.. MTAOS architecture and design considerations are also discussed in `tstn-026 <https://tstn-026.lsst.io/>`__.
+
+This purpose of this tech note is to describe the logic behind the design, and how the commands and variables are intended to be used.
+We only focus on the force related commands and variables for M1M3 and M2, and position related commands and variables for the hexapods.
+There are a lot more commands and varialbes for each CSC that are not covered here.
+
+####
+M1M3
+####
+
+On the top level, the M1M3 actuator forces, including x, y, and z-forces, fall into four categories.
+
+#. Applied forces, including forces applied by the AOS and other forces applied by the user during testing. The command to apply the forces is ``cmd_applyForces``. The input to the command is an 1D array of size 156. Each element of the array is a z-force values to be applied to a specific actuator. The actuators are ordered with ascending IDs, as found `here <https://github.com/lsst-ts/ts_m1m3supportgui/blob/develop/FATABLE.py/>`__. Every time the user-supplied forces are applied, there is an event ``evt_appliedForces`` which publishes the forces being applied. The input forces are aggregate forces, instead of forces on top of the existing forces. Therefore, a new ``cmd_applyForces`` command overrides all previous ``cmd_applyForces`` commands.
+
```

Comment:
I  am not sure I understand the link here for the actuators IDs ... do I need to know what the tables fatable means?